cinema/views.py

```diff
@@ -0,0 +1,46 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=200)
```

Comment:
I think best practise is use or status=200 or status=status.HTTP_200_OK in all fucntion. Don't mix this.

Reply:
Thank you :) I agree with u!